java-tools/OsmAndServer/src/main/resources/application.yml

```diff
@@ -86,9 +86,14 @@ spring:
       max-request-size: ${SPRING_SERVLET_MULTIPART_MAX_REQUEST_SIZE:50MB}
       enabled: true
   jpa:
-    properties.hibernate.dialect: org.hibernate.dialect.PostgreSQLDialect
-    properties.hibernate.jdbc.lob.non_contextual_creation: true
-    hibernate.ddl-auto: update
+    properties:
```

Comment:
why update shorter form, they are equal implementation so shorter form easier to red